index.js

```diff
@@ -1250,20 +1250,21 @@
       rng_pool = new Array();
       rng_pptr = 0;
       var t;
-      if(typeof window !== "undefined" && window.crypto) {
-        if (window.crypto.getRandomValues) {
-          // Use webcrypto if available
-          var ua = new Uint8Array(32);
-          window.crypto.getRandomValues(ua);
-          for(t = 0; t < 32; ++t)
-            rng_pool[rng_pptr++] = ua[t];
-        }
-        else if(navigator.appName == "Netscape" && navigator.appVersion < "5") {
-          // Extract entropy (256 bits) from NS4 RNG if available
-          var z = window.crypto.random(32);
-          for(t = 0; t < z.length; ++t)
-            rng_pool[rng_pptr++] = z.charCodeAt(t) & 255;
-        }
+      var crypto = window.crypto || window.msCrypto;
+		if(typeof window !== "undefined" && crypto) {
+		  if (crypto.getRandomValues) {
+			// Use webcrypto if available
+			var ua = new Uint8Array(32);
+			crypto.getRandomValues(ua);
+			for(t = 0; t < 32; ++t)
+			  rng_pool[rng_pptr++] = ua[t];
+		  }
+		  else if(navigator.appName == "Netscape" && navigator.appVersion < "5") {
+			// Extract entropy (256 bits) from NS4 RNG if available
+			var z = window.crypto.random(32);
+			for(t = 0; t < z.length; ++t)
+			  rng_pool[rng_pptr++] = z.charCodeAt(t) & 255;
+		  }
```

Comment:
You'll need to fix the indentation here (there should only be a few lines changed here).